app/src/main/java/org/thoughtcrime/securesms/database/Storage.kt

```diff
@@ -1542,6 +1546,34 @@ open class Storage(
         }
     }
 
+    /**
+     * This will create a control message used to indicate that you have accepted a message request
+     */
+    override fun insertMessageRequestResponseFromYou(threadId: Long){
+        val userPublicKey = getUserPublicKey() ?: return
+
+        val mmsDb = DatabaseComponent.get(context).mmsDatabase()
+        val message = IncomingMediaMessage(
```

Comment:
If we converted `IncomingMediaMessage` to Kotlin we could have named arguments, because that constructor is absolutely grim and invites mistakes!

Reply:
Yeah that is a nice update that we can work on post strings